src/main/java/transforms/ArithmeticTransform.java

```diff
@@ -0,0 +1,112 @@
+package transforms;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.openlattice.shuttle.dates.DateTimeHelper;
+import com.openlattice.shuttle.transformations.Transformation;
+import com.openlattice.shuttle.util.Constants;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+
+public class ArithmeticTransform extends Transformation<Map<String, String>> {
+    private String               operator;
+    private List<Transformation> leftTransforms, rightTransforms;
+    private              Double alternative;
+    private static final Logger logger = LoggerFactory.getLogger( DateTimeHelper.class );
+
+    /**
+     * Represents a transformation from string to datetime.
+     *
+     * @param leftTransforms:  transformations for left side of computation
+     * @param rightTransforms: transformations for right side of computation
+     * @param operator:        operator, one of +, -, /, *
+     * @param alternative:     if left or right transformation fail to parse to double: what should the value be (defaults to null);
+     */
+    public ArithmeticTransform(
+            @JsonProperty( Constants.LEFTTRANSFORMS ) List<Transformation> leftTransforms,
+            @JsonProperty( Constants.RIGHTTRANSFORMS ) List<Transformation> rightTransforms,
+            @JsonProperty( Constants.OPERATOR ) String operator,
+            @JsonProperty( Constants.ELSE ) Optional<Double> alternative
+    ) {
+        this.operator = operator;
+        this.leftTransforms = leftTransforms;
+        this.rightTransforms = rightTransforms;
+        this.alternative = alternative.orElse( null );
+    }
+
+    @JsonProperty( Constants.OPERATOR )
+    public String getOperator() {
+        return operator;
+    }
+
+    @JsonProperty( Constants.LEFTTRANSFORMS )
+    public List<Transformation> getLeftTransforms() {
+        return leftTransforms;
+    }
+
+    @JsonProperty( Constants.RIGHTTRANSFORMS )
+    public List<Transformation> getRightTransforms() {
+        return rightTransforms;
+    }
+
+    public Double getValueFromTransform( Object transformed ) {
+
+        String transformedString = Objects.toString( transformed, "" );
```

Comment:
Not blocking: to be clear if you get an object that is not a string then it will be converted to a string via it's .toString() method as opposed to returning this.alternative. Is this the desired behavior?

Reply:
Added this:
```
      if ( transformed instanceof Number ) {
            return (double) transformed;
        } else {
```
to avoid double transform.